vscode/polar-lang/syntaxes/polar.tmLanguage.json

```diff
@@ -0,0 +1,119 @@
+{
+  "$schema": "https://raw.githubusercontent.com/martinring/tmlanguage/master/tmlanguage.json",
+  "name": "Polar",
+  "patterns": [
+    { "include": "#inline-query" },
+    { "include": "#rule-def-head" },
+    { "include": "#rule-def-body" },
+    { "include": "#comment" }
+  ],
+  "repository": {
+    "inline-query": {
+      "begin": "(\\?=)",
+      "end": "(;)",
+      "beginCaptures": {
+        "1": { "name": "keyword.control" }
+      },
+      "patterns": [{ "include": "#expression" }]
+    },
+    "rule-def-head": {
+      "begin": "[\\s]*([\\w][\\w-]*)(\\()",
+      "end": "(\\))",
+      "beginCaptures": {
+        "1": { "name": "support.function" }
+      },
+      "patterns": [{ "include": "#string" }, { "include": "#number" }]
+    },
+    "rule-def-body": {
+      "begin": "(if)",
+      "end": "(;)",
+      "beginCaptures": {
+        "1": { "name": "keyword.control" }
+      },
+      "patterns": [{ "include": "#expression" }]
+    },
+    "expression": {
+      "patterns": [
+        { "include": "#comment" },
+        { "include": "#string" },
+        { "include": "#number" },
+        { "include": "#and" },
+        { "include": "#or" },
+        { "include": "#not" },
+        { "include": "#new" },
+        { "include": "#unify" },
+        { "include": "#in" },
+        { "include": "#forall" },
+        { "include": "#matches" }
```

Comment:
`cut`? `debug`? `print`?

Reply:
Debug and print are just builtin predicates so seems sorta odd to have them a different color. Cut, now that there are no parens makes sense though.

Reply:
Okay. FWIW, in the syntax highlighting I made for the website, `debug` gets turned green with or without parens, and `print` gets turned green when it has parens

Reply:
ok, added those